scripts/variantstore/variant_annotations_table/GvsValidateVAT.wdl

```diff
@@ -129,7 +136,7 @@ workflow GvsValidateVat {
                        SubpopulationMax.pass,
                        SubpopulationAlleleCount.pass,
                        SubpopulationAlleleNumber.pass,
-                       ClinvarSignificance.pass,
+                       DuplicateAnnotations.pass,
```

Comment:
Did you mean to replace `ClinvarSignificance` with `DuplicateAnnotations`? If so, you should get rid of the task itself.

Reply:
not yet---I want to check with Lee about how to handle this:

https://broadworkbench.atlassian.net/browse/VS-635